main/memcached/APKBUILD

```diff
@@ -1,29 +1,41 @@
 # Contributor: Jeff Bilyk <jbilyk@alpinelinux.org>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=memcached
-pkgver=1.5.5
-pkgrel=1
+pkgver=1.5.6
+pkgrel=0
 pkgdesc="Distributed memory object caching system"
 url="http://memcached.org/"
 arch="all"
 license="BSD"
-makedepends="cyrus-sasl-dev libevent-dev libseccomp-dev linux-headers"
+depends_dev="${pkgname}=${pkgver}-r${pkgrel}"
+makedepends="$depends_dev cyrus-sasl-dev libevent-dev libseccomp-dev linux-headers"
 install="$pkgname.pre-install"
 subpackages="$pkgname-dev $pkgname-doc"
 source="http://www.memcached.org/files/${pkgname}-${pkgver}.tar.gz
+	$pkgname.seccomp-musl.patch
 	$pkgname.confd
 	$pkgname.initd"
-builddir="${srcdir}/${pkgname}-${pkgver}"
+builddir="$srcdir/$pkgname-$pkgver"
 
 build() {
+	# extstore is broken on 32-bit
+	# https://github.com/memcached/memcached/issues/319
+	local _extstore=''
+	case "$CARCH" in
```

Comment:
Use `[ "$CARCH" != "x86" ] && _extstore="--enable-extstore"` instead.

Reply:
No, `[ "$CARCH" = "x86" ] || _extstore="--enable-extstore"` is better. Your variant would be interpreted as a failure on some shells (with `set -e` enabled).

Reply:
The `case` should be easiest to read and maintain going forward. What's the argument against using case here?

Reply:
I agree that `case` is a bit better in this case. My reaction was to @kaniini whose suggestion is actually worse, at least in terms of compatibility (and in this case it could be written in more compatible way that does not feel unnaturally).

I just don’t see how totally unnecessary left parenthesis (`  (x86) _extstore=…`) improves readability; it was the first time ever I’ve seen left parenthesis in `case` and I even wouldn’t know that it’s a valid shell syntax if I did not write a shell parser some time ago (and so studied POSIX Shell Command Language in detail).

Reply:
I had the opposite reaction that you seem to be expressing. I always thought `foo)` was very odd looking and was quite happy to find out that `(foo)` was valid syntax too.

Do you know of any other usages of the `foo)` syntax other than `case` in `C` based languages?